server/neptune/gateway/event/check_run_handler.go

```diff
@@ -13,6 +14,8 @@ import (
 
 const RequestedActionType = "requested_action"
 
+var checkRunRegex = regexp.MustCompile("atlantis/deploy: (?P<name>.+)")
```

Comment:
You don't have to do it as part of this, but we should probably also aggregate usage of this status into some sort of serializer struct that handles both serlalization and deserialization